azure-ipam/logger/logger.go

```diff
@@ -1,61 +1,42 @@
 package logger
 
 import (
-	"strings"
-
 	"github.com/pkg/errors"
 	"go.uber.org/zap"
 	"go.uber.org/zap/zapcore"
+	"gopkg.in/natefinch/lumberjack.v2"
 )
 
 type Config struct {
-	Level            string
-	OutputPaths      string // comma separated list of paths
-	ErrorOutputPaths string // comma separated list of paths
+	Level       string // Debug by default
+	Filepath   	string // if Empty log into <processname>-lumberjack.log within os.TempDir()
+	MaxSize    	int    // megabytes
```

Comment:
It will not be clear to me reading or writing the config that this is megabytes. We should either use the base unit, or rename the config appropriately.

Reply:
I suggest changing it to MaxSizeInMB.